nilearn/_utils/testing.py

```diff
@@ -585,12 +615,31 @@ def generate_group_sparse_gaussian_graphs(
     return signals, precisions, topology
 
 
-def skip_if_running_nose(msg=None):
+def skip_if(skip, skip_msg=None):
```

Comment:
I removed all the commits related to this, and opted for a more simple fix (ignore the warning), with a comment explaining.